src/rt/dmain2.d

```diff
@@ -550,6 +550,36 @@ private void formatThrowable(Throwable t, scope void delegate(in char[] s) nothr
     }
 }
 
+private auto parseExceptionOptions()
+{
+    static struct ExceptionOptions
+    {
+        bool trap;
+
+    @nogc nothrow:
+
+        bool initialize()
+        {
+            trap = rt_trapExceptions;
+            import core.internal.parseoptions : initConfigOptions;
+            return initConfigOptions(this, this.errorName);
+        }
+
+        void help()
+        {
+            string s = "Exception options are specified as whitespace separated assignments:
+    trap:0|1      - 0 disables trapping of exceptions, 1 exceptions will be trapped (default: %d)
+";
```

Comment:
This looks a bit excessive for a single option. Maybe just use rt_configOption? Or are you anticipating more exception options?

Reply:
Ok changed to `rt_configOption`. The downside is that now it can't re-use the existing `parse` code, but OTOH the CLI is shorter and easier.